releasenotes/notes/oracle-enrich-sqltext-aa778da34a8eb4ed.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Improve SQL text collection for self-managed installations. We are selecting text from `V$SQL` instead of `V$SQLSTATS`. If it isn't possible to query the text, we try to identify the context, such as parsing, or closing cursor, and put it in the SQL text.
```

Comment:
Is "we" the Agent? If so, I'd say the "Agent" instead of "we". :) Just makes it clearer.

```suggestion
    Improve SQL text collection for self-managed installations. We are selecting text from `V$SQL` instead of `V$SQLSTATS`. If it isn't possible to query the text, we try to identify the context, such as parsing or closing cursor, and put it in the SQL text.
```

Reply:
> Is "we" the Agent? If so, I'd say the "Agent" instead of "we". :) Just makes it clearer.

Thank you. Yes it's the Agent. I changed the text accordingly.